docs/src/main/sphinx/connector/pulsar.rst

```diff
@@ -0,0 +1,306 @@
+=================
+Pulsar connector
+=================
+
+.. toctree::
+    :maxdepth: 1
+    :hidden:
+
+    Tutorial <pulsar-tutorial>
+
+Overview
+--------
+
+`Apache Pulsar <https://pulsar.apache.org/docs/en/next/standalone/>`_ is a
+cloud-native, distributed messaging and streaming platform. 
+
+The Pulsar connector allows you to query data from Pulsar topics as tables in
+Trino:
+
+* A Pulsar cluster is configured as a catalog in Trino.
+
+* The Pulsar namespace or tenant combination is mapped to a schema in Trino.
+
+* A Pulsar topic schema is mapped to a Trino table schema.
+
+* A message in a Pulsar topic is presented as a row in Trino.
+
+For how to get started with the Pulsar connector, see the
+:doc:`pulsar-tutorial`.
+
+Requirements
+------------
+
+To connect to Pulsar, you need:
+
+* Pulsar 2.8.0 or higher.
+
+* Network access from the Trino coordinator and workers to the Pulsar web
+service API. 8080 is the default port.
```

Comment:
you need to indent this so that "Network" and "service" line up.. same applies to all other list items..

Reply:
@mosabua thanks for your instruction! I've indented and successfully previewed the contents:

![image](https://user-images.githubusercontent.com/50226895/124725457-74c60400-df3f-11eb-9308-bee481c6cced.png)

Any other comments on this doc PR?